src/compas_model/scene/elementobject.py

```diff
@@ -80,13 +77,18 @@ def __init__(
         self.show_edges = show_edges
         self.show_faces = show_faces
 
+        for child in self.element.children:
+            child_kwargs = kwargs.copy()
+            child_kwargs["item"] = child
+            self.add(**child_kwargs)
+
     @property
     def element(self) -> Element:
-        return self._element
+        return self.item
 
     @element.setter
     def element(self, element: Element) -> None:
-        self._element = element
+        self.item = element
         self._transformation = None
```

Comment:
We should perhaps consider make this read-only and remove such setters. Essentially bind `Object` with their `Item` throughout the entire life cycle

Reply:
remove it yes